auth/signin.html

```diff
@@ -28,9 +28,17 @@ <h1 class="hidden">Linkbrary</h1>
           <div class="auth-input email">
             <label for="id"> 이메일 </label>
             <div data-state="normal" class="input-wrapper">
-              <input id="id" name="id" placeholder="이메일을 입력해주세요." />
+              <input
+                id="id"
+                name="id"
+                placeholder="이메일을 입력해주세요."
+                class="email-input"
+              />
             </div>
-            <small data-state="normal" class="input-message"></small>
+            <small
+              data-state="normal"
+              class="input-message email-error-message"
+            ></small>
```

Comment:
자식 요소가 들어가지 않는다면 self-closing 태그를 이용하는게 개발자의 의도(단독으로 쓰임)를 더 전달할 수 있을거 같네요~
```suggestion
            <small
              data-state="normal"
              class="input-message email-error-message"
            />
```